cal-itp/benefits
# Refactor: eligibility forms

Closes #1564

Gets us closer to #1525

## What this PR does

* Removes all the form-specific fields from `EligibilityVerifier` model: `form_blurb`, `form_sub_label`, etc.
* Adds a single field `form_class` that holds the name of a Python form class, adds a method to create an instance of this class
* Creates a form class for `MSTCourtesyCard` verification, moves all config there (out of migration)

## Testing

* Run `bin/init.sh`
* Verify with MST Courtesy Card flow
* Ensure no functional regressions
* Ensure no design regressions
* Ensure no copy regressions
